engine/lua/cloe/schedule.lua

```diff
@@ -217,6 +220,7 @@ function cloe.schedule_test(test)
     cloe.validate({ test = { test, "table" }})
     cloe.validate({
         desc = { test.desc, "string", true },
+        info = {test.info, "table", true},
```

Comment:
Consider using autoformat; otherwise make sure you are being consistent with the surrounding code.